pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [tool.codespell]
 check-filenames = true
 check-hidden = true
-ignore-words-list = "ot"
+ignore-words-list = "ot,gallary"
 # Remove .git and venv skips when integrated with pre-commit
 # Feel free to un-skip docs, examples, and experimental, you will just need to
 # work through many typos (--write-changes and --interactive will help)
-skip = "./.git,./*venv,./docs,./examples,./experimental,*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt"
+skip = "./.git,./*venv,./examples,./experimental,*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
```

Comment:
Can you update the comment to remove `docs` from the list?

Reply:
Yeah sure, will do in the next commit